yt_idefix/data_structures.py

```diff
@@ -583,8 +593,8 @@ def _parse_definitions_header(self) -> None:
                 expr = re.sub(r"CONST_\w+", self._get_constants, expr)
                 expr = re.sub(r"UNIT_(\w+)", self._get_unit, expr)
                 expr = re.sub(r"sqrt", "np.sqrt", expr)
-                expr = re.sub(constexpr, self._get_constants, expr)
-                self.parameters[unit] = eval(expr)
+                expr = re.sub(_CONST_REGEXP, self._get_constants, expr)
```

Comment:
Ah sorry, I just noticed that I forgot to remove this line in my previous PR. It's the same as line#593 and now I prefer to make it explicit here.

Reply:
good catch, thanks !